README.aspnet.md

```diff
@@ -49,6 +49,12 @@ After the application starts, navigate to `http://localhost:8000` in your web br
 
 # Full Tag Listing
 
+## Windows Server, version 1909 amd64 Tags
+Tag | Dockerfile
+---------| ---------------
+4.8-20191008-windowsservercore-1909, 4.8-windowsservercore-1909, 4.8, latest | [Dockerfile](https://github.com/microsoft/dotnet-framework-docker/blob/master/4.8/aspnet/windowsservercore-1909/Dockerfile)
```

Comment:
Just want to make sure that the time stamp will rev at release time?

Reply:
Yes, I'll have a separate PR for servicing changes that will rev the timestamp.